### PR TITLE
VZ-3340 - Kiali Acceptance test

### PR DESCRIPTION
**Pull request by pfmackin**
Initial acceptance test for Kiali.  This also handles the upgrade case, where Kiali is not installed for 1.0.  The PR includes 2 minor Network Policies changes because the kiali label name changed.

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tests/e2e/verify-install/kiali/kiali_test.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+		client, kialiErr = k8sutil.GetKubernetesClientset()
+		Expect(kialiErr).ToNot(HaveOccurred())
+		httpClient, kialiErr = pkg.GetSystemVmiHTTPClient()
+		Expect(kialiErr).ToNot(HaveOccurred())
+		kialiSupported, kialiErr = pkg.IsVerrazzanoMinVersion("1.1.0")
+		Expect(kialiErr).ToNot(HaveOccurred())
+	})
+
+	// It Wrapper to only run spec if Kiali is supported on the current Verrazzano installation
+	WhenKialiInstalledIt := func(description string, f interface{}) {
```
Comment (michael-cico) [inline tests/e2e/verify-install/kiali/kiali_test.go:45]: This is good, we should generalize this for the flaky test support.

@agv426 @fredtibbitts WDYT?  Might be good way to wrap flaky specs without disabling the whole suite.

**Review by michael-cico — APPROVED** [review]

File: tests/e2e/verify-install/kiali/kiali_test.go (hunk 1 above)
Comment (anders-swanson) [reply tests/e2e/verify-install/kiali/kiali_test.go:45]: Yeah, good for flaky tests, make a `WhenRunFlakyTestsIt` that uses some environment variable or flag param at the package level to determine runtime eligibility.

Comment (anders-swanson) [reply tests/e2e/verify-install/kiali/kiali_test.go:45]: alt. is to split the specs into separate files.

Comment (michael-cico) [reply tests/e2e/verify-install/kiali/kiali_test.go:45]: Fred mentioned also that perhaps 2.0 might have better support for conditionalizing individual specs, but I haven't looked at that yet.

Comment (fredtibbitts) [reply tests/e2e/verify-install/kiali/kiali_test.go:45]: Gingko 2 has spec labeling and also a FlakeAttempts decoration, which allows you to specify how many times a spec should be retried if it fails. Labels give you fine-grained control over what suites and specs get run, so we can label individual specs as flaky. I think the plan is to move to Ginkgo 2 but not sure on the timeline.

**Pull request merged**